news/_posts/releases/2022-03-07-prestashop-1-7-8-4-maintenance-release.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.8.4"
+subtitle: "Maintenance version of the 1.7.8.x branch"
+date:   2022-03-07 18:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2022/01/1.7.8.4_banner.jpg
+icon: icon-checked
+tags:
+ - version
+ - patch
+ - releases
+ - 1.7.8.x
+ - 1.7
+
+---
+
+PrestaShop 1.7.8.4 is now available. This maintenance release fixes 23 bugs and feature request.
+
+![1.7.8.4 is available!](/assets/images/2022/01/1.7.8.4_banner.jpg)
+
+
+We suggest upgrading your store quickly to benefit from these bug fixes. Make sure to perform a full manual backup before, so you can roll back the upgrade if something goes wrong.
+
+Reminder: the latest version of the 1-Click Upgrade module is v4.14.0, don't forget to upgrade it.
+
+## Bug fixes and improvements
+
+- [Reverts the integration of Symfony kernel in the webservice dispatcher](https://github.com/PrestaShop/PrestaShop/issues/27411)
+- [Update links to docs : docs.prestashop-project.org instead of doc.prestashop.com](https://github.com/PrestaShop/PrestaShop/issues/27493)](https://github.com/PrestaShop/PrestaShop/issues/27509)
```

Comment:
```suggestion
- [Update links to docs : docs.prestashop-project.org instead of doc.prestashop.com](https://github.com/PrestaShop/PrestaShop/issues/27493)
```